tests/js/client/shell/shell-keygen.js

```diff
@@ -873,10 +897,12 @@ function PersistedLastValueSuite () {
           assertEqual(0, lastValue);
 
           for (let i = 0; i < 10; ++i) {
-            c.insert({}); 
+            c.insert({});
             p = c.properties();
             let newLastValue = p.keyOptions.lastValue;
-            assertTrue(newLastValue > lastValue);
+            if (Number(numShards) === 2) {
```

Comment:
Why this `if` condition` here? Shouldn't the assertion in the next line always hold true, for both `padded` and `autoincrement`?